planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -70,17 +70,22 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner(const rclcpp::NodeOptions & n
   // interface publisher
   traj_pub_ = create_publisher<Trajectory>("~/output/path", 1);
   virtual_wall_pub_ = create_publisher<MarkerArray>("~/virtual_wall", 1);
+  odom_path_pub_ = create_publisher<Path>
+          ("/planning/scenario_planning/lane_driving/motion_planning/path", 1);
 
   // interface subscriber
   path_sub_ = create_subscription<Path>(
     "~/input/path", 1, std::bind(&ObstacleAvoidancePlanner::onPath, this, std::placeholders::_1));
+
   odom_sub_ = create_subscription<Odometry>(
     "~/input/odometry", 1, [this](const Odometry::SharedPtr msg) { ego_state_ptr_ = msg; });
 
   // debug publisher
   debug_extended_traj_pub_ = create_publisher<Trajectory>("~/debug/extended_traj", 1);
   debug_markers_pub_ = create_publisher<MarkerArray>("~/debug/marker", 1);
   debug_calculation_time_pub_ = create_publisher<StringStamped>("~/debug/calculation_time", 1);
+  debug_traj_pub_ = create_publisher<Trajectory>
+          ("/planning/scenario_planning/lane_driving/motion_planning/debug_trajectory", 1);
```

Comment:
Please use the same topic name format as others (e.g. `~/debug/traj` or similar) and remap it like the others.

Reply:
agree.

Reply:
agree.